mc/Mongo-Core.package/SCRAMAuthMechanism.class/class/authenticateAgainstAdmin.st

```diff
@@ -0,0 +1,3 @@
+accessing
+authenticateAgainstAdmin
+	overrideDB := true
```

Comment:
Now, I realize this method has no senders and uses a variable that is not defined.
I will remove it.
@rydnr

Reply:
done, in master